### ISSUE TITLE
Morphological sliders

### ISSUE THREAD

**Issue by SiberDrac** [Freedom-of-Form-Foundation/anatomy3d#14]
Determine a minimal set of dimensions a given bone (here, humerus) should have available as sliders a user could adjust, e.g. length, girth, curvature, etc.

**Comment by AdamNorberg** [comment]
Building on top of this - something I've been thinking about is that at some point we'll need to make sure models users create can be packed into files, which means planning how to represent them on disk. The numbers mentioned here are probably the data to save.

**Comment by Lathreas** [comment]
@AdamNorberg Indeed! We are currently thinking of using JSON to store the data, since it's a nice and standard all-round format that should work well for our use-case. The exact layout of it should still be finalized, though. Let me turn it into a separate issue so any comments on the save file format can be put there!
